src/ui/units/collections/components/CollectionContent/CollectionContent.tsx

```diff
@@ -343,7 +344,7 @@ export const CollectionContent = React.memo<Props>(
                                     workbookId: item.workbookId,
                                     workbookTitle: item.title,
                                     initialCollectionId: item.collectionId,
-                                    onApply: refreshContent,
+                                    onApply: refreshPage,
```

Comment:
It's not related to the current task, but we can use `onApply: refreshContent` in dialogs `DIALOG_MOVE_COLLECTION` and `DIALOG_MOVE_WORKBOOK`. This will reduce amount of unnecessary requests.